doc/classes/LineEdit.xml

```diff
@@ -301,6 +307,12 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="editing_toggled">
+			<param index="0" name="toggled_on" type="bool" />
+			<description>
+				Emitted when the [LineEdit] switches between focus and edit modes.
```

Comment:
This isn't clear, since edit mode insures focus, maybe `select and edit modes` or `focus and focus-edit modes` is better.

Reply:
`Emitted when the [LineEdit] switches in or out of edit mode.`?
I'd also explain edit mode in the class' description.